content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md

```diff
@@ -11,7 +11,7 @@ versions:
 
 {% tip %}
 
-**Tip:** You can also apply a label in the Labels drop-down menu within an issue or pull request.  
+**Tip:** A user can apply up to a maximum of 100 labels to issues and pull requests.
```

Comment:
```suggestion
**Tip:** You can apply a maximum of 100 labels to issues and pull requests.
```

To keep it consistent and friendly, we like refer to readers in the second person 🥰

Reply:
Thanks! Great to know it :)

Reply:
```suggestion
**Tip:** You can apply a maximum of 100 labels to issues and pull requests.
```

So sorry, I don't think I was clear! You can make these edits in the PR after a review and then I'll get notified that your PR has been changed and I can review it again and approve. 😄  Try merging this suggested change and pushing it up to your branch. ⚡